_includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-feature.md

```diff
@@ -243,6 +247,57 @@ Make sure that the OMS transition diagram shows the expected transitions:
 
 {% endinfo_block %}
 
+3. Configure the Push Notification provider:
+
+**src/Pyz/Zed/PickingListPushNotification/PickingListPushNotificationConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\PickingListPushNotification;
+
+use Spryker\Zed\PickingListPushNotification\PickingListPushNotificationConfig as SprykerPickingListPushNotificationConfig;
+
+class PickingListPushNotificationConfig extends SprykerPickingListPushNotificationConfig
+{
+    /**
+     * @uses \Spryker\Zed\PushNotificationWebPushPhp\PushNotificationWebPushPhpConfig::WEB_PUSH_PHP_PROVIDER_NAME
+     *
+     * @var string
+     */
+    protected const PUSH_NOTIFICATION_PROVIDER_NAME = 'web-push-php';
+
+    /**
+     * @return string
+     */
+    public function getPushNotificationProviderName(): string
+    {
+        return static::PUSH_NOTIFICATION_PROVIDER_NAME;
+    }
+}
+```
+
+4. Configure the `User` module to execute post save plugins: // TODO: check if this one is correct
```

Comment:
* This part has to be moved to the 'Spryker Core Back Office' integration guide.